.travis.yml

```diff
@@ -8,3 +8,5 @@ script:
   - yarn lint
   - yarn test
 dist: trusty
+notifications:
```

Comment:
You are correct, this actually sends a notification for every successful travis build, which means every commit

The logic to filter this is actually set in Samson. Here is a screenshot of that configuration page:

![image](https://user-images.githubusercontent.com/1471573/65513112-848f6800-ded2-11e9-809d-64f5ed59a133.png)

As you can see, it says a merge to master creates a release and also triggers a deploy on that release

I prefer this because if the release creation fails it still is deployed to staging where tests can be run against it. and we can later on fix the creation of release

If you look at this screenshot:
![image](https://user-images.githubusercontent.com/1471573/65513254-d9cb7980-ded2-11e9-8eaf-1a7e35cc435d.png)
Here Samson actually received a webhook callback but decided that no deploy should be triggered.